Documentation/cluster-crd.md

```diff
@@ -80,11 +81,31 @@ A Placement configuration is specified (according to the kubernetes [PodSpec](ht
 - `podAntiAffinity`: kubernetes [PodAntiAffinity](https://kubernetes.io/docs/api-reference/v1.6/#podantiaffinity-v1-core)
 - `tolerations`: list of kubernetes [Toleration](https://kubernetes.io/docs/api-reference/v1.6/#toleration-v1-core)
 
+### Cluster-wide Resources Configuration Settings
+
+**WARNING** Before setting resource requests/limits, please take a look at the Ceph documentation for recommendations for each component: [Ceph - Hardware Recommendations](http://docs.ceph.com/docs/master/start/hardware-recommendations/).
+
+Set the resource requests/limits for rook components:
+- `api`: Set resource requests/limits for the API, see [Resource Requirements/Limits](#resource-requirementslimits).
```

Comment:
since the `resource requirements/limits` section is right after this, I don't think you need to link to it from these four bullets.